book.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'rental'
+
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rentals(date)
+    rental = Rental.new(date)
+    @rentals << rental
+    rental.book = self
+    rental
```

Comment:
- You have not properly implemented the `has-many` association between the `book` and `rental` class. Please fix this 🙏 . I expect you would have something like this:
```ruby
def add_rentals(date, person)
  Rental.new(date, self, person)
end
```